src/assets/styles/loading.css

```diff
@@ -1,276 +0,0 @@
-.empty-space::before {
```

Comment:
yeah .empty-space was extra but there was some dup styles. We don't need all those browser prefixes for keyframes, so we're good without it.